.github/workflows/pypi-build.yml

```diff
@@ -0,0 +1,44 @@
+on:
```

Comment:
Can we name the workflow `pypi-publish`?

Reply:
yep